packages/optimizer/spec/transformers/valid/ReorderHeadTransformer/supports_esm/expected_output.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <link rel="preload" href="https://cdn.ampproject.org/v0.css" as="style">
+  <link rel="stylesheet" href="https://cdn.ampproject.org/v0.css">
+  <script async src="https://cdn.ampproject.org/v0.mjs" type="module" crossorigin="anonymous"></script>
+  <script async nomodule src="https://cdn.ampproject.org/v0/amp-experiment-0.1.js" custom-element="amp-experiment"></script>
+  <script async custom-element="amp-experiment" src="https://cdn.ampproject.org/v0/amp-experiment-0.1.mjs" type="module" crossorigin="anonymous"></script>
+  <script async nomodule src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js" custom-template="amp-mustache"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.mjs" type="module" crossorigin="anonymous"></script>
+  <link href="https://example.com/favicon.ico" rel="icon">
+  <link href="https://fonts.googleapis.com/css?foobar" rel="stylesheet" type="text/css">
+  <link as="script" crossorigin="anonymous" href="https://cdn.ampproject.org/v0.mjs" rel="modulepreload">
```

Comment:
This needs to be moved up along with the `preload` as well. Ref https://github.com/ampproject/amp-toolbox-php/pull/130/commits/a3a99fd5c66dad645a1ab47138d000b32b635df4

Reply:
Thanks for the patience taking it up with my empty brain today...